pennylane/capture/meta_type.py

```diff
@@ -124,6 +165,53 @@ def _(*_, **__):
     return primitive
 
 
+def create_measurement_obs_primitive(
+    measurement_type: type, name: str
+) -> Optional["jax.core.Primitive"]:
+    """Create a primitive corresponding to the input type where the abstract inputs are an operator."""
+    if not has_jax:
+        return None
+
+    primitive = jax.core.Primitive(name)
+
+    @primitive.def_impl
+    def _(obs, **kwargs):
+        return measurement_type(obs=obs, **kwargs)
+
+    abstract_type = _get_abstract_measurement()
+
+    @primitive.def_abstract_eval
+    def _(*_, **__):
+        abstract_eval = measurement_type._abstract_eval  # pylint: disable=protected-access
+        return abstract_type(abstract_eval, n_wires=None)
+
+    return primitive
+
+
+def create_measurement_wires_primitive(
```

Comment:
Very nice: We can pass a list of wires as argument to a function and pass that list on to `qml.probs`, for example.

A bit unintuitive: The unpacking of wires into single integer-valued arguments is passed on to the jaxpr call signature. Probably we will handle this under the hood in an execution pipeline, but I did not quite see this one coming :D

Reply:
One of the reason I'm currently invetigating capturing wires into it's own primitive and abstract class.  Currently investigating, so will have more to say soon.